src/lib/ngsi/AttributeDomainName.cpp

```diff
@@ -99,11 +99,11 @@ void AttributeDomainName::present(const std::string& indent)
 {
   if (string != "")
   {
-    PRINTF("%sAttributeDomainName: %s\n", indent.c_str(), string.c_str());
+    LM_F(("%sAttributeDomainName: %s", indent.c_str(), string.c_str()));
```

Comment:
Is LM_F always printed or only in debug compilation mode?

Is LM_F using the log format as defined in https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide#Log_format? 

Which msg level is used (I understand it should be INFO)?